apps/src/applab/AppLabCrosshairOverlay.jsx

```diff
@@ -1,5 +1,6 @@
 /** @file App Lab-specific Crosshair Overlay */
 
+import React from 'react';
```

Comment:
After moving the unit tests into the test/unit folder and loading test/unit/**/*.js as tests, I believe the run order for the unit tests subtly changed.  The unit tests that shallow-render this overlay started failing because this component was unable to find `React` when it was loaded.  For some reason, our `setExternalGlobals()` helper was not fixing the problem.

I'm suspicious that the way module loading works in our transpiled ES6 has something to do with this, but I can't prove it.

[We `browserify-global-shim` React](https://github.com/code-dot-org/code-dot-org/blame/staging/apps/package.json#L31) in our apps build as of #7181 (this means any time we require/import `'react'` we'll retrieve `window.React` instead of loading a module).  Conveniently, we don't global-shim in tests so this actually loads React, removing the need to set up globals.  I propose that, now that we have it shimmed, we should switch to loading React this way everywhere instead of treating it as a global (even though it is provided as one in our real build) - I think @wjordan pointed out that we should be doing this a long time ago, and @pcardune set it up to support the app lab bundle (which also does not shim React).

That said, I'm happy to spend some more time trying to wrangle the globals here if you don't want to introduce this change now.